dadab/dadab.js

```diff
@@ -3,77 +3,101 @@ based on:
  - Simple Image Glitching Processing - datamoshing.com | 2016
  - a pixel sorting sketch from Jerome Martinez | 2017
  */
- 
+
 // output needs to be 1066 x 800
- 
+
+var capture;
+var gif;
 var img;
-var imgFileName = "vera";
-var fileType = "jpg";
+var imgFileName = 'vera';
+var fileType = 'jpg';
+
+var START_FRAME = 15;
+var END_FRAME = 115;
+
+const serverUrl = 'http://localhost:3000';
+
+const socketOptions = {
+  transports: ['websocket'],
+  'force new connection': true
+};
+const socket = io(serverUrl, socketOptions);
 
-function preload() {
+function preload () {
   // load an image from our library to set the canvas createCanvas
-  img = loadImage(imgFileName + "." + fileType);
+  img = loadImage(imgFileName + '.' + fileType); // eslint-disable-line no-undef
 }
 
-
-function setup() {
-  
-  // use only numbers (not variables) for the createCanvas() command, Processing 3
-  var canvas = createCanvas(img.width, img.height);
-  canvas.parent("p5canvas");
+function setup () {
+  var canvas = createCanvas(img.width, img.height); // eslint-disable-line no-undef
+  canvas.parent('p5canvas');
 
   // allows the viewer to recreateCanvas and update surface to image file dimensions
   // surface.setResizable(true);
   // surface.setSize(img.width, img.height);
-  
+
+  // todo: tweak these settings
+  capture = createCapture(VIDEO);
```

Comment:
I think some of this `capture` business can be removed or tweaked. It just came from a gist I found [linked in the README.md file further down].